app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" translatable="false">ICSx⁵</string>
```

Comment:
Strings with `translatable="false"` should simply be omitted. They will be taken from the original `strings.xml`.